sechub-cli/src/daimler.com/sechub/cli/constants.go

```diff
@@ -1,10 +1,18 @@
 // SPDX-License-Identifier: MIT
+
 package cli
 
 // DefaultSecHubConfigFile represents the name of the sechub configuration file
 // used per default when no other set
 const DefaultSecHubConfigFile = "sechub.json"
 
+// DefaultZipExcludeDirPatterns - Define directory patterns to exclude from zip file:
+// code in directories named "test" is not considered to end up in the binary
+// also ignore .git directory
+var DefaultZipExcludeDirPatterns = []string{"**/test/**", "**/.git/**"}
```

Comment:
Why not use `const` here?

Reply:
I have tried that already. `const` for []string ist not allowed.
go-lint says: _"const initializer []string literal is not a constant. No quick fixes available"_

This type is needed or things like
`append(json.CodeScan.Excludes, DefaultZipExcludeDirPatterns...)`
will not work. (both need to be of type []string)